app/scripts/modules/core/src/api/network.interceptor.ts

```diff
@@ -72,6 +75,16 @@ export class NetworkInterceptor implements IHttpInterceptor {
         }, (retryCount + 1 + Math.random()) * 1000);
       });
     }
+
+    if (status === 401 && SETTINGS.feature.iapRefresherEnabled) {
+      $http.get('/_gcp_iap/do_session_refresh').then(() => {
+        return this.$q.resolve(this.$injector.get('$http')(config)).then((result: any) => {
```

Comment:
I think the type of `result` is `T`, not `any`

Reply:
Done